src/store/drawing/__tests__/reducer-test.ts

```diff
@@ -47,11 +49,24 @@ describe('store/drawing/reducer', () => {
         });
     });
 
+    describe('setDrawingStrokeAction', () => {
+        test('should set drawing stroke', () => {
+            const newState = reducer(state, setDrawingStrokeAction({ color: '#000' }));
+
+            expect(newState).toMatchObject({
+                stroke: {
+                    color: '#000',
+                    size: 4,
+                },
+            });
+        });
+    });
+
     describe('toggleAnnotationModeAction', () => {
-        test('should reset to initial state', () => {
+        test('should reset to initial state expect stroke', () => {
             const newState = reducer(getDirtyState(), toggleAnnotationModeAction(Mode.DRAWING));
 
-            expect(newState).toEqual(initialState);
+            expect(newState).toEqual({ ...initialState, stroke: { color: '#000', size: 1 } });
```

Comment:
Is it possible for the size of the stroke to change?

Reply:
If preview emits `stroke` with `size` property, yes